studies/test_recording_method.py

```diff
@@ -135,44 +151,63 @@ def test_external_researcher_preview(self):
         self.assertIsNone(Response.objects.get(uuid=response_uuid).recording_method)
 
 
-@skip("Need AWS credentials.")
 class RecordingMethodEFPTestCase(TestCase):
     def test_efp(self):
         study_type = StudyType.get_ember_frame_player()
-        user, study, child = get_user(study_type)
+        _, study, child = get_user(study_type)
         response = Response.objects.create(
             study_type=study_type, child=child, study=study
         )
         self.assertEqual(response.recording_method, "pipe")
 
```

Comment:
For the EFP test case, I was going to suggest that we add tests for the web/external view and researcher preview view (like you've done with external and jsPsych study types. But then I realized that the EFP recording method comes from the response POST, so I assume that's why we don't need a test for it here?

Reply:
"Need" is a strong word. I did this to be complete, and the only way I could confirm that EFP was setting the recording method correctly.  The inference is that if EFP uses the API to generate the Response and  EFP relies on the default of the `recording_method` field, then this makes sure that the default setting remains "pipe."

Reply:
Testing for how EFP creates and uses the response should be in the EFP test cases.